### PR TITLE
fix: active id should update selected tab

### PR DESCRIPTION
**Pull request by eljefe223**
# Description
Changing the `activeid` attribute now update the selected tab.

Closes: #3925 

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [x] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

File: packages/web-components/fast-foundation/src/tabs/tabs.ts [hunk 1]
```diff
@@ -47,6 +47,16 @@ export class Tabs extends FASTElement {
...
     public activeid: string;
+    public activeidChanged(): void {
+        if (
+            this.$fastController.isConnected &&
+            this.tabs.length <= this.tabpanels.length
+        ) {
+            this.setTabs();
+            this.setTabPanels();
+            this.handleActiveIndicatorPosition();
+        }
```
Comment (chrisdholt) [inline packages/web-components/fast-foundation/src/tabs/tabs.ts:58]: Can you add a couple tests showing that updating the activeId changes it accordingly?

Comment (chrisdholt) [reply packages/web-components/fast-foundation/src/tabs/tabs.ts:58]: I believe there is a test for setting the activeId - we should add one after for updating.

Comment (eljefe223) [reply packages/web-components/fast-foundation/src/tabs/tabs.ts:58]: Updated

**Review by EisenbergEffect — APPROVED** [review]

File: packages/web-components/fast-foundation/src/tabs/tabs.ts [hunk 2]
```diff
@@ -47,6 +47,16 @@ export class Tabs extends FASTElement {
      */
     @attr
     public activeid: string;
```
Comment (nicholasrice) [inline packages/web-components/fast-foundation/src/tabs/tabs.ts:49]: I believe this should have an `@internal` tsdoc flag unless this is intended to be used by authors.

**Review by SethDonohue — APPROVED** [review]

File: packages/web-components/fast-foundation/src/tabs/tabs.ts (hunk 2 above)
Comment (eljefe223) [reply packages/web-components/fast-foundation/src/tabs/tabs.ts:49]: Why would this be internal? Its an attribute authors can write to.

**Pull request merged**